homework3/hw3/Helpers/SportType.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  SportType.swift
+//  hw3
+//
+//  Created by Tina Jureško on 27.03.2025..
+//
+
+import Foundation
+import UIKit
+
+enum SportType {
+    case football
+    case basketball
+    case americanFootball
+    
+    var title: String {
+        switch self {
+        case .football:
+            return "Football"
+        case .basketball:
+            return "Basketball"
+        case .americanFootball:
+            return "American Football"
+        }
+    }
+    
+    var icon: UIImage? {
+        switch self {
+        case .football:
+            return UIImage(named: "icon_football")
```

Comment:
Images su isto definirati u Assetima kao i Colors. Ajde omogući za Image assete isto safe dohvat kao što si omogućila za boje. Pa ćeš moći dohvatiti image sa `.iconFootball`.